data/disney

```diff
@@ -47,10 +47,18 @@ disney.sg
 20thcenturystudios.com.au
 20thcenturystudios.com.br
 20thcenturystudios.jp
+adobedtm.com
 adventuresbydisney.com
 babble.com
 babyzone.com
+bamgrid.com
+bam.nr-data.net
 beautyandthebeastmusical.co.uk
+braze.com
+cdn.optimizely.com
+cdn.registerdisney.go.com
+cws.conviva.com
```

Comment:
This is a domain owned by Conviva that offering solutions for online video optimization and online video analytics. Can it prove that this domain only provides services to Disney?